configs/dev-genesis.js

```diff
@@ -38,10 +38,12 @@ config.express.session.prefix = 'veres-one.';
 config.mongodb.name = 'veres_one_localhost_genesis';
 config.mongodb.host = 'localhost';
 config.mongodb.port = 27017;
-config.mongodb.local.collection = 'veres_one_localhost_genesis';
-config.mongodb.username = 'veres';
-config.mongodb.password = 'password';
-config.mongodb.adminPrompt = true;
+// mongodb auth options are commented out, because if a username
+// and password are set it tries to login.
+//config.mongodb.username = 'veres';
+//config.mongodb.password = 'password';
+//config.mongodb.connectOptions.authSource = 'admin';
+//config.mongodb.adminPrompt = true;
```

Comment:
The default for code that isn't used is to remove it. If it sticks around in commented form, we need a comment that explains both why it's commented and why it wasn't just removed. The above comment just makes me wonder:

1. Why don't we want it to try to login? Is that temporary?
2. Is it supposed to try and login sometimes and I may want these lines here to more easily accomplish that?
3. How is this situation different for veres-one vs. other bedrock apps -- such that these lines should be here as commented? (Why the inconsistency, what's special?)

So, can you add the reason why these were not removed? Same here as for any other commented code.

Reply:
I would leave it there in case someone wants to try a development build with auth enabled.

Reply:
Great! If you add that as a comment to the code I think it would answer my questions.

Reply:
comments were added addressing this.